testing/bdd/test_passive_node_shall_not_modify_documents.py

```diff
@@ -31,8 +60,6 @@ def when_processes_document(distributing_node, test_context, template_file):
 
 @then('the emitted document is identical to the received one')
 def then_emitted_document_identical(distributing_node, test_context):
-    from ebu_tt_live.utils import compare_xml
     prod_carriage = distributing_node.producer_carriage
     assert prod_carriage.emit_data.call_count == 1
     assert compare_xml(test_context['document'], prod_carriage.emit_data.call_args[0][0])
```

Comment:
Really minor point, but using `args[0][0]` is less readable and less robust than `args[1]['data']` as on line 51.